app/views/includes/govuk_wrapper.scala.html

```diff
@@ -52,14 +57,14 @@
 
 @govukLayout(
     pageTitle = Some(pageTitle),
-    headBlock = Some(hmrcHead(head)),
-    headerBlock = Some(hmrcStandardHeader(
-        serviceUrl = None,
-        serviceName = Some(messages("service.name")),
-        displayHmrcBanner = false
-    )),
+    headBlock = Some(hmrcInternalHead(head, nonce = CSPNonce.get)),
```

Comment:
<img width="2838" alt="Screenshot 2022-12-01 at 12 40 23" src="https://user-images.githubusercontent.com/11138469/205073141-0f860dff-1666-4d9e-bb17-7680d335be2a.png">
not sure if that is expected header but I doubt that :D

Reply:
think it should be internal header as it is an internal stride service, but yeah need to fix the url

Reply:
okay just an update should now nav back to the dashboard

https://user-images.githubusercontent.com/47248104/205640044-cff24fbe-24d4-47fa-ad31-1a78d7e46efa.mov

Reply:
fixed